web/ui/react-app/src/pages/LabelsTable.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Button, Badge, Table } from 'reactstrap';
+
+interface LabelProps {
+  value: Record<string, Record<string, string>>[];
+  name: string;
+}
+
+const printLabels = (labels: Record<string, string>) => {
+  return Object.entries(labels).map(([key, value]) => {
+    return (
+      <div key={key}>
+        <Badge color="primary" className="mr-1">
+          {`${key}="${value}"`}
+        </Badge>
+      </div>
+    );
+  });
+};
+
+export const LabelsTable: FC<RouteComponentProps & LabelProps> = ({ value, name }) => {
+  const [showMore, doToggle] = useState(false);
```

Comment:
More customary for `useState` result names would be `showMore` and `setShowMore` (`doToggle` is a bit misleading because it doesn't toggle by itself).